Sources/PagecallSDK/NativeBridge.swift

```diff
@@ -13,7 +13,7 @@ enum BridgeEvent: String, Codable {
 }
 
 enum BridgeAction: String, Codable {
-    case createSession, start, stop, getPermissions, requestPermission, pauseAudio, resumeAudio, getAudioDevices, setAudioDevice, requestAudioVolume
+    case createSession, initController, start, stop, getPermissions, requestPermission, pauseAudio, resumeAudio, getAudioDevices, setAudioDevice, requestAudioVolume, dispose
```

Comment:
"init"은 사용할 수 없는 예약어라서 "initController" 를 대신 사용했습니다.

Reply:
[Optional]
```suggestion
    case createSession, `init`, start, stop, getPermissions, requestPermission, pauseAudio, resumeAudio, getAudioDevices, setAudioDevice, requestAudioVolume, dispose
```

이렇게 가능할것 같아요
https://stackoverflow.com/a/31315650